webapp/pages/admin/users.vue

```diff
@@ -1,24 +1,23 @@
 <template>
-  <div>
-    <ds-space>
-      <ds-card :header="$t('admin.users.name')">
-        <ds-form v-model="form" @submit="submit">
-          <ds-flex gutter="small">
-            <ds-flex-item width="90%">
-              <ds-input
-                model="query"
-                :placeholder="$t('admin.users.form.placeholder')"
-                icon="search"
-              />
-            </ds-flex-item>
-            <ds-flex-item width="30px">
-              <base-button filled circle type="submit" icon="search" :loading="$apollo.loading" />
-            </ds-flex-item>
-          </ds-flex>
-        </ds-form>
-      </ds-card>
-    </ds-space>
-    <ds-card v-if="User && User.length">
+  <div class="admin-users">
```

Comment:
could this be `article`?

Reply:
I'd probably go for `section` rather than `article` but either would work. However, I'd prefer to not have this element there at all! That would have required bigger changes, though and at some point I just had to stop myself because this PR was already getting way too big... I'll keep it for next time, ok? :grin: